action.yml

```
@@ -14,11 +14,8 @@ inputs:
   access_secret:
     description: "access_secret"
     required: true
-  build1:
-    description: "build1"
-    required: true
-  build2:
-    description: "build2"
+  architecture:
+    description: "architecture"
```

The "description" should describe what this is and what it accepts. Supported values can be cross checked with launchpad snap recipes.

```suggestion
  architecture:
    description: "Target build architecture, such as: amd64, arm64, armhf"
```